### PR TITLE
Some streamlining of the next plugin

### PR DESCRIPTION
**Pull request by tboerger**

### CODE REVIEW

**Comment by jackspirou** [comment]
Hi @tboerger, thanks for all the updates across the plugins :) i'm assuming the description you provided for https://github.com/drone-plugins/drone-webhook/pull/5 would basically be the same for this pull request.

My one question is why we are using `go get` to query and download the dep every time we produce a build?  Do we no longer want to lock vendor dep versions for plugins on the whole?

Thanks for getting me up to speed and LGTM!

**Comment by jackspirou** [comment]
LGTM

**Comment by tboerger** [comment]
@jackspirou yes, as you can see on the commit messages it's pretty similar.

For the vendoring I talked to brad on gitter where we said that our dependencies are so simple that we can skip the vendoring.

File: Dockerfile [hunk 1]
```diff
@@ -1,9 +1,15 @@
...
+#     make deps build
 #     docker build --rm=true -t plugins/drone-hipchat .
 
-FROM gliderlabs/alpine:3.2
-RUN apk-install ca-certificates
+FROM alpine:3.2
+
+RUN apk update && \
+  apk add \
+    ca-certificates && \
```
Comment (nlf) [inline Dockerfile:11]: same nit, this can be `apk --update add ca-certificates`

**Comment by nlf** [comment]
again, aside from the one nit LGTM

**Pull request merged**